tests/draft3/type.json

```diff
@@ -9,7 +9,47 @@
                 "valid": true
             },
             {
-                "description": "a float is not an integer",
+                "description": "a negative integer is an integer",
+                "data": -1,
+                "valid": true
+            },
+            {
+                "description": "zero is an integer",
+                "data": 0,
+                "valid": true
+            },
+            {
+                "description": "negative zero is an integer",
+                "data": -0,
+                "valid": true
+            },
+            {
+                "description": "a float with zero fractional part is an integer",
+                "data": 1.0,
+                "valid": true
+            },
```

Comment:
Not sure why this wasn't included in `draft3` already, but there might be a reason!